blog/resources/views/reset/reset.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+use App\Auth\Form\ResetForm;
+use Yiisoft\Form\Field;
+use Yiisoft\Html\Html;
+use Yiisoft\Html\Tag\Form;
+use Yiisoft\Router\UrlGeneratorInterface;
+use Yiisoft\Translator\TranslatorInterface;
+use Yiisoft\View\WebView;
+
+/**
+ * @var WebView               $this
+ * @var TranslatorInterface   $translator
+ * @var UrlGeneratorInterface $urlGenerator
+ * @var string                $csrf
+ * @var ResetForm             $formModel
+ */
+$this->setTitle($translator->translate('layout.reset'));
+?>
+
+<div class="container py-5 h-100">
+    <div class="row d-flex justify-content-center align-items-center h-100">
+        <div class="col-12 col-md-8 col-lg-6 col-xl-5">
+            <div class="card border border-dark shadow-2-strong rounded-3">
+                <div class="card-header bg-dark text-white">
+                    <h1 class="fw-normal h3 text-center"><?= Html::encode($this->getTitle()) ?></h1>
+                </div>
+                <div class="card-body p-5 text-center">
+                    <?= Form::tag()
+                        // note: the reset function actually appears in the ResetController
+                        ->post($urlGenerator->generate('auth/reset'))
+                        ->csrf($csrf)
+                        ->id('resetForm')
+                        ->open() ?>
+
+                    <?= Field::text($formModel, 'login')->addInputAttributes(['value'=> $login ?? '', 'readonly'=>'readonly']) ?>
```

Comment:
Shouldn't user enter login in case current user is guest?

Reply:
Yes. The reason why the login field is pulled into the resetForm and is readonly is that the user, with my system mentioned above, is already authenticated ie. not guest, and is authorized ie. has certain permissions via console assignRole. The input attribute displays their current  login username specific to self and is therefore readonly.

It would be nice to have this option incorporated in the main package as an alternative to the current login/reset norm/method.